src/app/pages/client-reviews/client-reviews.component.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { mockReviewsResponse, reviewResponse } from 'src/app/shared/mocks';
+import { ClientReviewsService } from 'src/app/services/client-reviews/client-reviews.service';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { httpClientSpy, toastServiceSpy, reviewsSpy, resetSpies } from 'src/app/helpers/spies';
+import { of, throwError } from 'rxjs';
+import { ToastrService } from 'ngx-toastr';
+import { RouterTestingModule } from '@angular/router/testing';
+import { ClientReviewsComponent } from 'src/app/pages/client-reviews/client-reviews.component';
+import { DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { NgxSpinnerModule } from 'ngx-spinner';
+
+describe('ClientReviewsComponent', () => {
+  let component: ClientReviewsComponent;
+  let fixture: ComponentFixture<ClientReviewsComponent>;
+  let debugElement: DebugElement;
+  const url = 'http://127.0.0.1:8000/api/v1/auth/1/reviews';
```

Comment:
we can use the url in the environment here
```suggestion
  const url = `${environment.api_url}/1/reviews`;
```